File: Taller Tipos de clases/C#/Clase Interfaz/Notificacion.cs
interface INotificacion
{
    void Enviar(string msg);
}

class Email : INotificacion
{
    public string Correo;

    public Email()
    {
        Correo = "example@mail.com";
    }

    public Email(string correo)
    {
        Correo = correo;
    }

    public Email(Email e)
    {
        Correo = e.Correo;
    }

    public void Enviar(string msg) { }
}

File: Taller Tipos de clases/JAVA/Clase Interfaz/Notificacion.java
interface Notificacion {
    void enviar(String msg);
}

class Email implements Notificacion {
    String correo;

    public Email() {
        correo = "example@mail.com";
    }

    public Email(String correo) {
        this.correo = correo;
    }

    public Email(Email e) {
        this.correo = e.correo;
    }

    public void enviar(String msg) {}
}

